src/Microsoft.SqlTools.ServiceLayer/LanguageServices/LanguageService.cs

```diff
@@ -293,15 +295,22 @@ public void InitializeService(ServiceHost serviceHost, SqlToolsContext context)
             }
         }
 
-// turn off this code until needed (10/28/2016)
-#if false
-        private static async Task HandleDefinitionRequest(
-            TextDocumentPosition textDocumentPosition,
-            RequestContext<Location[]> requestContext)
+        internal static async Task HandleDefinitionRequest(TextDocumentPosition textDocumentPosition, RequestContext<Location[]> requestContext)
```

Comment:
Or else if IsQuickInfoEnabled is false? There should be some config flag that skips processing

---
In reply to: [89247609](https://github.com/Microsoft/sqltoolsservice/pull/160#discussion_r89247609) [](ancestors = 89247609)